test/metadata/adapter.xml

```diff
@@ -272,7 +272,7 @@ under the License.
 <!--			<filename>../metadata/func/flatnetwork.xml</filename>-->
 <!--			<testname>Flat network test</testname>-->
 <!--		</test>-->
-	
+
```

Comment:
I would suggest removing all this excessive code in comment (maybe out of scope for this)